src/MeetingsSDKAdapter.js

```diff
@@ -79,24 +80,23 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
       sendShare: false,
     };
 
-    sdkMeeting
-      .getMediaStreams(mediaSettings)
-      .then(([localStream, localShare]) => {
-        sdkMeeting
-          .addMedia({
-            localShare,
-            localStream,
-            mediaSettings,
-          })
-          .catch((error) => {
-            // eslint-disable-next-line no-console
-            console.error(`Unable to add local media to meeting "${ID}"`, error);
-          });
-      })
-      .catch((error) => {
+    try {
+      const [localStream, localShare] = await sdkMeeting.getMediaStreams(mediaSettings);
+
+      try {
```

Comment:
Should we move this outside the `getMediaStreams` try block? I don't think it has to be nested does it?

Reply:
I had the blocks, to catch each errors respectively. base on my knowledge, each `catch` needs a `try`.

Reply:
I don't think it's necessary to have the nested `try`s to distinguish between errors. You could just have one `catch` for either error and just log out `Unable to add local media to meeting "${ID}"`` which is what this method is trying to do.

Reply:
My suggestion is to do the two `try`s separately:

```js
try {
 await getMediaStreams
} catch (error) {
}

try {
 await addMedia
} catch (error) {
```

Reply:
I ended up using one try block. good catch.

Reply:
That works. If we need to be more specific as to which function failed, we would need to do two try blocks.